applications/FluidDynamicsApplication/custom_utilities/rotating_frame_utility.h

```diff
@@ -0,0 +1,111 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main author:     Sebastian Ares de Parga Regalado
+//
+
+#pragma once
+
+// System includes
+#include <string>
+#include <iostream>
+
+// External includes
+#include "includes/mesh_moving_variables.h"
+
+// Include kratos definitions
+#include "includes/define.h"
+
+// Application includes
+#include "utilities/variable_utils.h"
+
+// Project includes
+#include "utilities/builtin_timer.h"
+#include "utilities/math_utils.h"
```

Comment:
```suggestion
// System includes
#include <string>
#include <iostream>

// External includes


// Project includes
#include "includes/define.h"
#include "includes/mesh_moving_variables.h"
#include "utilities/builtin_timer.h"
#include "utilities/math_utils.h"
#include "utilities/variable_utils.h"

// Application includes

```
To follow our standard include lists.